### PR TITLE
Action to read from SSM Param Store

### PR DESCRIPTION
**Pull request by IgorDobryn**
Action to retrieve information about parameters in a specific hierarchy

```yml
- uses: railsware/github-actions/read-from-ssm-param-store@master
  with:
    path: /sandbox/
    region: ew-west-1
```

Outputs:

- `param_json` - JSON read from param store.

### CODE REVIEW

**Comment by fuksito** [comment]
Is it intentional to end path with a slash?

**Comment by IgorDobryn** [comment]
> Is it intentional to end path with a slash?

yes. This is how it should be in ParamStore

**Review by fuksito — APPROVED** [review]

File: read-from-ssm-param-store/readFromParamStore.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  return data.Parameters;
+}
+
+module.exports = readFromParamStore;
+
+// async function run () {
+//   console.log(await readFromParamStore('/sandbox/deploy/', 'eu-central-1'))
+// }
+//
+// run()
```
Comment (fuksito) [inline read-from-ssm-param-store/readFromParamStore.js:57]: Do we need it to remain for easier debug?

Comment (IgorDobryn) [reply read-from-ssm-param-store/readFromParamStore.js:57]: Yup

**Comment by IgorDobryn** [comment]
Merging it to ublock upcoming PR. If any issues will adress them post merge

**Pull request merged**